src/components/molecules/CategoryButton/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Styled from 'styled-components';
+import Button from 'components/atoms/Button';
+
+interface IProps {
+  tag: string;
+  isMore?: boolean;
+  className?: string | undefined;
+  ref?: React.RefObject<HTMLInputElement>;
+  onClickInterest?: ((tag: string) => void) | undefined;
+}
+interface IData {
+  readMore?: boolean;
+}
+
+function CategoryButton({ tag, isMore, onClickInterest }: IProps): React.ReactElement {
+  const onSelectInterest = () => {
+    onClickInterest && onClickInterest(tag);
+  };
+  return (
+    <SCategoryButton readMore={isMore}>
+      <Button onClick={onSelectInterest} className="button">
+        {tag}
+      </Button>
+    </SCategoryButton>
+  );
+}
+
+const SCategoryButton = Styled.label`
```

Comment:
```suggestion
const SCategoryButton = Styled.label<IData>`
```
로 바꿔주면 밑에서 정의 안해도 됩니다!